services/net/transcription/TranscriptionManager.cs

```diff
@@ -228,14 +228,21 @@ private async Task UpdateTranscriptionAsync(ContentModel content)
         // TODO: Handle different storage locations.
         // Remote storage locations may not be easily accessible by this service.
         var path = content.FileReferences.FirstOrDefault()?.Path;
-        var safePath = Path.Join(_options.VolumePath, path.MakeRelativePath());
+        var safePath = Path.Join(_options.FilePath, path.MakeRelativePath());
+
+        var mediaType = await IsVideoAsync(safePath) ? SourceMediaType.Video : SourceMediaType.Audio;
+        if (mediaType == SourceMediaType.Video)
```

Comment:
Move this within the `if (File.Exists(safePath)` statement.

Reply:
I moved this session out of the **if (File.Exists(safePath)** since i overwrote the safe path after the convert. I'll do another if statement to check whether converted file exists or not if it make more sense to move it inside **if (File.Exists(safePath))**

Reply:
You only overwrite the `safePath` if (a) it's a video, and (b) the convert is successful.